README.md

```diff
@@ -1,83 +1,64 @@
-
-For more information about contributing to the Kubernetes documentation, see:
-
-* [Start contributing](https://kubernetes.io/docs/contribute/start/)
-* [Staging Your Documentation Changes](http://kubernetes.io/docs/contribute/intermediate#view-your-changes-locally)
-* [Using Page Templates](http://kubernetes.io/docs/contribute/style/page-templates/)
-* [Documentation Style Guide](http://kubernetes.io/docs/contribute/style/style-guide/)
-* [Localizing Kubernetes Documentation](https://kubernetes.io/docs/contribute/localization/)
-
-## Localization `README.md`'s
-|  |  |
-|---|---|
-|[French README](README-fr.md)|[Korean README](README-ko.md)|
-|[German README](README-de.md)|[Portuguese README](README-pt.md)|
-|[Hindi README](README-hi.md)|[Spanish README](README-es.md)|
-|[Indonesian README](README-id.md)|[Chinese README](README-zh.md)|
-|[Japanese README](README-ja.md)|[Vietnamese README](README-vi.md)|
-|[Russian README](README-ru.md)|[Italian README](README-it.md)|
-|[Polish README](README-pl.md)|[Ukrainian README](README-uk.md)|
-|||
-
-## Running the website locally using Docker
-
-The recommended way to run the Kubernetes website locally is to run a specialized [Docker](https://docker.com) image that includes the [Hugo](https://gohugo.io) static website generator.
-
-> If you are running on Windows, you'll need a few more tools which you can install with [Chocolatey](https://chocolatey.org). `choco install make`
+## Running the website locally using Hugo
 
-> If you'd prefer to run the website locally without Docker, see [Running the website locally using Hugo](#running-the-website-locally-using-hugo) below.
+See the [official Hugo documentation](https://gohugo.io/getting-started/installing/) for Hugo installation instructions. Make sure to install the Hugo extended version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
 
-If you have Docker [up and running](https://www.docker.com/get-started), build the `kubernetes-hugo` Docker image locally:
+To run the website locally when you have Hugo installed:
 
 ```bash
-make docker-image
+git clone https://github.com/kubernetes/website.git
+cd website
+hugo server
```

Comment:
Not to block a merge with this feedback _but_ I notice we're gradually losing the information that people can run Hugo in a container. 

That's a good thing to document IMO, containers are definitely a convenient way to run this kind of tooling. @jimangel where do you think that that detail should live?

Reply:
My thought behind this was dropping some of the abstractions (ie: using a makefile to run a command or building a container to host the docs). Hugo is supported on the 3 major platforms so compatibility isn't really an issue.

The main goal here is lowering the bar for future contributors to get started on their journey. IMO the `README.md` should have a single, simple, approach to getting started rather than `/shrug` pick what you want.

I think the folks who want to run Hugo in a container most likely don't need the instructions hosted on k/websites' `README.md`. Also, Hugo doesn't have an official container which means the new contributor has to build it themselves. I would be more apt to leave this content in if folks could clone and execute `docker run --rm -it -v $(pwd):/src -v $(pwd)/public:/target hugo-official:version-tag` or similar.

As far as a solution, I would be for any of the following:
- removing the Docker content entirely
- adding an alternate doc in k/website root repo
- moving to [CONTRIBUTING.md](https://github.com/kubernetes/website/blob/master/CONTRIBUTING.md)
- migrating the content to [kubernetes.io](https://kubernetes.io/docs/contribute/)

Thoughts?

Reply:
> - migrating the content to kubernetes.io

That's my favorite.

Reply:
I'll open an issue up tonight when I'm wrangling for this content to be created.

Reply:
Hi folks, @jimangel can you point here issue what you created, please?
Ask because we'd like sync README.md in our loclization (#20718) but I also would like keep Docker section somewhere, in same manner how it was did for EN version.